techbuzz/src/main/java/com/sivalabs/techbuzz/users/domain/services/UserService.java

```diff
@@ -66,4 +72,12 @@ public void verifyEmail(String email, String token) {
         user.setVerificationToken(null);
         userRepository.updateVerificationStatus(user);
     }
+
+    @CacheEvict(cacheNames = "user", allEntries = true)
```

Comment:
Why to evict cache while fetching a user info?

Reply:
apology it was a  copy-paste typo removed